client/my-sites/plans/jetpack-plans/jetpack-complete-page/components/cta-buttons/style.scss

```diff
@@ -0,0 +1,40 @@
+.jetpack-complete-page__cta-buttons {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+	& > * {
+		flex: 1;
+	}
+	@include breakpoint-deprecated( "<960px" ) {
+		flex-direction: column;
+	}
+
+	.jetpack-complete-page__get-complete-button {
+		@include breakpoint-deprecated( ">960px" ) {
+			margin-right: 2em;
+		}
```

Comment:
Since we're using `inline-flex`, there's a more suitable property we can use to keep the buttons separated: `gap`. I see in the original designs there's a spacing of 24px between the two (we generally avoid `em`, for... reasons), so something like this:

```scss
// starting in the scope of __cta-buttons
@include breakpoint-deprecated( '>960px' ) {
	gap: 24px;
}

.jetpack-complete-page__get-complete-button {
	// remove breakpoint rule for margin-right
	// ...
```